sbt

```diff
@@ -26,7 +26,11 @@ do
 done
 
 java $DEBUG_PARAMS \
-    -Xms1024M -Xmx2048M -Xss1M -XX:+CMSClassUnloadingEnabled -XX:MaxPermSize=1024M \
+    -Xms1024M -Xmx2048M \
+    -Xss1M \
+    -XX:+CMSClassUnloadingEnabled \
+    -XX:MaxPermSize=1024M \
     $CONF_PARAMS \
     $TC_PARAMS \
+    -Dsbt.log.noformat=true \
```

Comment:
I think we only want this flag on TeamCity? Coloured output on our own shells is extremely useful!

Reply:
I don't know any way I can pass this in to the sbt invocation generated by snyk inside npm.
I could check for a TC-specific env var?  That's getting a bit ugly though.